KubeArmor/monitor/systemMonitor.go

```diff
@@ -118,6 +120,9 @@ type ContextCombined struct {
 // StopChan Channel
 var StopChan chan struct{}
 
+//go:embed system_monitor.*
```

Comment:
We should nams these binaries different so they are differentiated from runtime compiled binaries in general

Reply:
I've changed the binary names to `embedded_system_monitor` to reflect that this would be embedded inside the Kubearmor binary